lib/src/room.dart

```diff
@@ -1688,11 +1688,14 @@ class Room {
 
   /// Requests a missing [User] for this room. Important for clients using
   /// lazy loading. If the user can't be found this method tries to fetch
-  /// the displayname and avatar from the profile if [requestProfile] is true.
+  /// the displayname and avatar from the server if [requestState] is true.
+  /// If that fails, it falls back to requesting the global profile if
+  /// [requestProfile] is true.
   Future<User?> requestUser(
     String mxID, {
     bool ignoreErrors = false,
     bool requestProfile = true,
+    bool requestState = true,
```

Comment:
nitpick but `requestState` for requesting a profile from server sounds like a really confusing name for that variable. Can you make this a bit more verbose?